#! /bin/sh /usr/share/dpatch/dpatch-run
## svn-042-3842_x-content-mimetype.dpatch by Tim Retout <tim@retout.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch from b.g.o. 510325 by Stephane Delcroix adding an f-spot-import
## DP: .desktop file for appropiate x-content mimetypes.

@DPATCH@
diff -urNad f-spot~/Makefile.am f-spot/Makefile.am
--- f-spot~/Makefile.am	2008-04-15 21:08:14.000000000 +0100
+++ f-spot/Makefile.am	2008-04-15 21:08:15.000000000 +0100
@@ -36,7 +36,8 @@
 
 desktop_in_files=	\
 	f-spot.desktop.in	\
-	f-spot-view.desktop.in
+	f-spot-view.desktop.in	\
+	f-spot-import.desktop.in
 
 desktop_files=$(desktop_in_files:.desktop.in=.desktop)
 @INTLTOOL_DESKTOP_RULE@
diff -urNad f-spot~/configure.in f-spot/configure.in
--- f-spot~/configure.in	2008-04-15 21:08:14.000000000 +0100
+++ f-spot/configure.in	2008-04-15 21:08:15.000000000 +0100
@@ -316,4 +316,5 @@
 f-spot.spec
 f-spot.desktop.in
 f-spot-view.desktop.in
+f-spot-import.desktop.in
 ])
diff -urNad f-spot~/f-spot-import.desktop.in.in f-spot/f-spot-import.desktop.in.in
--- f-spot~/f-spot-import.desktop.in.in	1970-01-01 01:00:00.000000000 +0100
+++ f-spot/f-spot-import.desktop.in.in	2008-04-15 21:08:15.000000000 +0100
@@ -0,0 +1,20 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+_Name=F-Spot Photo Manager
+_GenericName=Photo Manager
+_Comment=Import into F-Spot
+NoDisplay=true
+Exec=f-spot-import %u
+Icon=f-spot
+MimeType=x-content/image-dcf;x-content/image-picturecd
+StartupNotify=true
+Terminal=false
+Type=Application
+Categories=Graphics;Photography;GNOME;GTK;
+X-GNOME-Bugzilla-Bugzilla=GNOME
+X-GNOME-Bugzilla-Product=f-spot
+X-GNOME-Bugzilla-Component=General
+X-GNOME-Bugzilla-Version=@VERSION@
+X-GNOME-DocPath=f-spot/f-spot.xml
+X-SuSE-translate=true
